### DISCUSSION TITLE
RDS with OLED SSD1306

### DISCUSSION THREAD

**Discussion by z34pec** [pu2clr/RDA5807#13] [Ideas]
Hello.
Is there a way to implement RDS on this ( RDA5807_07_NANO_OLED_V2.ino ) sketch with SSD1306 display driver?
It is good to put it instead of pin7 - bass.
Thank you

**Comment by pu2clr** [comment]
I'll check it out.
Ricardo.

On Mon, Jul 17, 2023 at 6:17 PM z34pec ***@***.***> wrote:

> Hello.
> Is there a way to implement RDS on this ( RDA5807_07_NANO_OLED_V2.ino )
> sketch with SSD1306 display driver?
> It is good to put it instead of pin7 - bass.
> Thank you
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pu2clr/RDA5807/discussions/13>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAVSJNAJDS24FQ4QL4OK2G3XQWTUZANCNFSM6AAAAAA2NPXNK4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 

QRZ: http://www.qrz.com/db/PU2CLR
Skype: ricardo_caratti
Mobile: +55 61 981442296
